core/src/syn/parser/stmt/define.rs

```diff
@@ -581,6 +581,8 @@ impl Parser<'_> {
 		let mut res = DefineTableStatement {
 			name,
 			permissions: Permissions::none(),
+			// Default to ANY if not specified in the DEFINE statement
+			kind: TableType::Any,
```

Comment:
Actually `Tabletype::Any` is the default value for `TableType`. Do we need to be explicit?